README.md

```diff
@@ -389,6 +389,11 @@ Upgrade example:
 python scripts/test_install_xcpng.py 10.0.0.2 f0f5f010-80c6-25ae-44a2-1fb154e32d14 upgrade 8.2.1
 ```
 
+Restore example:
+```
+python scripts/test_install_xcpng.py 10.0.0.2 f0f5f010-80c6-25ae-44a2-1fb154e32d14 restore 8.2.1
+```
+
```

Comment:
```suggestion
Restore example:
```
python scripts/test_install_xcpng.py 10.0.0.2 f0f5f010-80c6-25ae-44a2-1fb154e32d14 restore 8.2.1
```
Note: in case of restore, the version must be that of the installer (here 8.2.1), not the version of XCP-ng that will be restored.

```